src/api-handlers/mines.ts

```diff
@@ -51,7 +52,7 @@ const getMines = async (fastify, request) => {
     } else {
       query.land_id = land_id;
     }
-    console.log(query);
+    // console.log(query);
```

Comment:
nitpick, but may be time to start removing this kind of stuff.wdyt

Reply:
agreed, pls remove commented out unused code / debug fragments

Reply:
removed